ui/components/component-library/text/text.js

```diff
@@ -98,7 +98,19 @@ export const Text = React.forwardRef(
 
 Text.propTypes = {
   /**
-   * The variation of font styles including sizes and weights of the Text component (display, heading, body)
+   * The variation of font styles including sizes and weights of the Text component
+   * Possible values:
+   * `DISPLAY_MD` LG Screen: 48px / Sm Screen: 32px,
+   * `HEADING_LG` LG Screen: 32px / Sm Screen: 24px,
+   * `HEADING_MD` LG Screen: 24px / Sm Screen: 18px,
+   * `HEADING_SM` LG Screen: 18px / Sm Screen: 16px,
+   * `BODY_LG_MEDIUM` LG Screen: 18px / Sm Screen: 16px,
+   * `BODY_MD` LG Screen: 16px / Sm Screen: 14px,
+   * `BODY_MD_BOLD` LG Screen: 16px / Sm Screen: 14px,
+   * `BODY_SM` LG Screen: 14px / Sm Screen: 12px,
+   * `BODY_SM_BOLD` LG Screen: 14px / Sm Screen: 12px,
+   * `BODY_XS` LG Screen: 12px / Sm Screen: 10px,
```

Comment:
Can we weave in some familiar formatting using the screen size mixins?

```suggestion
   * `DISPLAY_MD` base font-size: 32px / screen-md-min { font-size: 48px }
   * `HEADING_LG` base font-size: 24px / screen-md-min { font-size: 32px  }
   * `HEADING_MD` base font-size: 18px / screen-md-min { font-size: 24px  }
   * `HEADING_SM` base font-size: 16px / screen-md-min { font-size: 18px  }
   * `BODY_LG_MEDIUM` base font-size: 16px / screen-md-min { font-size: 18px  }
   * `BODY_MD` base font-size: 14px / screen-md-min { font-size: 16px  }
   * `BODY_MD_BOLD` base font-size: 14px / screen-md-min { font-size: 16px  }
   * `BODY_SM` base font-size: 12px / screen-md-min { font-size: 14px  }
   * `BODY_SM_BOLD` base font-size: 12px / screen-md-min { font-size: 14px  }
   * `BODY_XS` base font-size: 10px / screen-md-min { font-size: 12px  }
```

Reply:
No not sure. Was just floating an idea. From the inspect view there is nothing to indicate Lg Screen / Sm Screen so was trying to tie it to something semi familiar like the breakpoint mixin

<img width="777" alt="Screen Shot 2022-11-22 at 10 01 55 AM" src="https://user-images.githubusercontent.com/8112138/203388483-2ac91bf1-c1f3-447d-83b0-bb02123e621b.png">